modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc

```diff
@@ -0,0 +1,162 @@
+----
++
+.Example output
+[source,terminal]
+----
+NAME     STATUS STATE                  CONSUMER            BMC                    HARDWARE PROFILE ONLINE ERROR
+master-1 OK     externally provisioned kni1-master-1       ipmi://172.22.0.1:6231                  true
+master-2 OK     externally provisioned kni1-master-2       ipmi://172.22.0.1:6232                  true
+master-3 OK     available                                  ipmi://172.22.0.1:6233 unknown          true
+worker-0 OK     provisioned            kni1-worker-0-kjpmp ipmi://10.19.143.29    dell-raid        true
+----
++
+. There are no `MachineSet` objects for control plane nodes, so you must create a `MachineSet` object instead. You can copy the image URL and checksum from the other BareMetalHost objects.
++
+[source,terminal]
+----
+$ cat <<EOF | oc apply -f -
+----
++
+.Example output
+[source,terminal]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: Machine
+metadata:
+  labels:
+	machine.openshift.io/cluster-api-cluster: kni1
+	machine.openshift.io/cluster-api-machine-role: master
+	machine.openshift.io/cluster-api-machine-type: master
+  name: kni1-master-3
+  namespace: openshift-machine-api
+spec:
+  providerSpec:
+	value:
+  	hostSelector: {}
+  	image:
+    	checksum: http://172.22.0.3:6180/images/rhcos-44.81.202004250133-0-openstack.x86_64.qcow2/rhcos-44.81.202004250133-0-compressed.x86_64.qcow2.md5sum
+    	url: http://172.22.0.3:6180/images/rhcos-44.81.202004250133-0-openstack.x86_64.qcow2/rhcos-44.81.202004250133-0-compressed.x86_64.qcow2
+  	userData:
+    	name: master-user-data
+EOF
+----
++
+[NOTE]
+====
+You can define the `BareMetalHost`, `Secret`, and `MachineSet` objects in a single YAML file, for example, `sample.yaml` and create the objects using the following command:
+
+[source,terminal]
+----
+$ oc create -f sample.yaml
+----
+====
++
+. The provisioning process (by using the baremetal-operator) installs RHCOS and prepares the host to be added to the cluster:
++
+[source,terminal]
+----
+$ oc get bmh -A
+----
++
+.Example output
+[source,terminal]
+----
+NAMESPACE               NAME           STATUS PROVISIONING STATUS     CONSUMER        BMC                     HARDWARE PROFILE   ONLINE   ERROR
+openshift-machine-api   kni1-master-0  OK     provisioned             kni1-worker-0   ipmi://10.19.136.22     dell-raid          true
+openshift-machine-api   master-1       OK     externally provisioned  kni1-master-1   ipmi://172.22.0.1:6231                     true
+openshift-machine-api   master-2       OK     externally provisioned  kni1-master-2   ipmi://172.22.0.1:6232                     true
+openshift-machine-api   master-3       OK     provisioning            kni1-master-3   ipmi://172.22.0.1:6233  unknown            true
+openshift-machine-api   worker-0       OK     provisioned             kni1-worker-1   ipmi://10.19.143.29     dell-raid          true
+----
++
+. The BareMetalHost is added to the cluster:
```

Comment:
Because this is a step, it should be an action, for example:
```suggestion
. Verify that the `BareMetalHost` object is added to the cluster:
```

Reply:
Updated as suggested.